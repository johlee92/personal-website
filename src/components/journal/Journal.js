import React, { Component } from 'react';
import './Journal.css';

class Journal extends Component {
    render() {
        return (
            <div>
            </div>
        )
    }
}

export default Journal;