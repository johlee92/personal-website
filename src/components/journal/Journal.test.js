import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import Journal from './Journal';

describe(`Journal component`, () => {
    it('renders without crashing', () => {
        const div = document.createElement('div');
        ReactDOM.render(
                <Journal />
            , div);
        ReactDOM.unmountComponentAtNode(div);
    });

    it('renders the component as expected', () => {
        const tree = renderer
            .create(
                <Journal />
            )
            .toJSON();
        expect(tree).toMatchSnapshot();  
    });
})
