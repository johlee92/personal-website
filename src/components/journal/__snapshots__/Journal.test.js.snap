// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Journal component renders the component as expected 1`] = `<div />`;
